### DISCUSSION TITLE
How to extend $env/dynamic/... on startup?

### DISCUSSION THREAD

**Discussion by vekunz** [sveltejs/kit#7372] [Q&A]
Hello,

I have a question regarding the $env/... packages of SvelteKit. Is it possible to extend the $env/dynamic/... lists on the startup of the SvelteKit app? We run our app with adapter-node in a Docker container. Our runtime mounts several files (not named .env or similar) into our docker container with credentials and similar stuff. Currently, we read these files and write the values in a custom data structure. This data structure can then be used from inside the server-side logic.

Now we are evaluating the $env mechanism of SvelteKit. It would be great if we could extend the $env/dynamic/... stuff of SvelteKit with our custom variables. Is this currently possible? If yes, how is this possible?

**Comment by j4w8n** [comment]
Based on https://github.com/sveltejs/kit/discussions/7383#discussioncomment-3963158, this is currently not possible.
